content/en/docs/admin/admission-controllers.md

```diff
@@ -21,7 +22,7 @@ is authenticated and authorized.  The controllers consist of the
 administrator. In that list, there are two special controllers:
 MutatingAdmissionWebhook and ValidatingAdmissionWebhook.  These execute the
 mutating and validating (respectively) [admission control
-webhooks](/docs/admin/extensible-admission-controllers/#external-admission-webhooks)
+webhooks](/docs/admin/extensible-admission-controllers.md#external-admission-webhooks)
```

Comment:
Please update the root from `/docs/` to `/content/en/docs/`, for this and any other links in the files involved in this PR.